.github/workflows/cd.yml

```diff
@@ -555,11 +576,16 @@ jobs:
       # This dependency can be skipped if gcs-only is true. In that case, this step
       # is still executed due to the "!(failure() || cancelled())" condition.
       - publish-to-catalog
+
+    # TODO: Temporarily disabled until IT re-enables the GitHub App.
+    #   See https://raintank-corp.slack.com/archives/C024NL6AEJH/p1745860508760709.
```

Comment:
nit: I'd remove the link (I am not sure what you want to mean with the thread anyway)

Reply:
Yeah these comments should go away once IT enables the GitHub app which was disabled. I did this in order to test in https://github.com/grafana/grafana-community-team/issues/382 or it would fail with a "This installation has been suspended" error. This must go away before merging. I have submitted the PR so that the majority of the changes can be reviewed and we can merge it quickly once the app is enabled by IT admins.

Reply:
okay, ignore my comments then regarding temporary changes. The rest LGTM 👍